cpu/cc2538/include/cc2538_rf.h

```diff
@@ -185,27 +195,83 @@ enum {
+                                  been updated at least once since RX was
+                                  started. Cleared when leaving RX.*/
+    demod_cca        = 0x0d, /**< Clear channel assessment. See FSMSTAT1
+                                  register for details on how to configure
+                                  the behavior of this signal. */
+    sampled_cca      = 0x0e, /**< A sampled version of the CCA bit from
+                                  demodulator. The value is updated whenever
+                                  a SSAMPLECCA or STXONCCA strobe is issued.*/
+    sfd_sync         = 0x0f, /**< Pin is high when a SFD has been received
+                                  or transmitted. Cleared when leaving
+                                  RX/TX respectively. Not to be confused
+                                  with the SFD exception.*/
+    tx_active        = 0x10, /**< Indicates that FFCTRL is in one of the TX
+                                  states. Active-high.*/
+    rx_active        = 0x11, /**< Indicates that FFCTRL is in one of the
+                                  RX states. Active-high. */
+    ffctrl_fifo      = 0x12, /**< Pin is high when one or more bytes are
+                                  in the RXFIFO. Low during RXFIFO overflow. */
+    ffctrl_fifop     = 0x13, /**< Pin is high when the number of bytes
+                                  in the RXFIFO exceeds the programmable
+                                  threshold or at least  one complete
+                                  frame is in the RXFIFO. Also highduring
+                                  RXFIFO overflow. Not to be confused with
+                                  the FIFOP exception.*/
+    packet_done      = 0x14, /**< A complete frame has been received.
+                                  I.e., the number of bytes set by the
+                                  frame-length field has been received.*/
+    rfc_xor_rand_i_q = 0x16, /**< XOR between I and Q random outputs.
+                                  Updated at 8 MHz.*/
+    rfc_rand_q       = 0x17, /**< Random data output from the Q channel
+                                  of the receiver. Updated at 8 MHz.*/
+    rfc_rand_i       = 0x18, /**< Random data output from the I channel
+                                  of the receiver. Updated at 8 MHz */
+    lock_status      = 0x19, /**< 1 when PLL is in lock, otherwise 0 */
+    pa_pd            = 0x20, /**< Power amplifier power-down signal */
+    lna_pd           = 0x2a, /**< LNA power-down signal*/
+    disabled         = 0xff, /**< disabled */
 };
 
+/** @} */
+
+/**
+ * @name    RF CORE observable signals settings
+ */
+#ifndef CONFIG_CC2538_RF_OBS_0
+#define CONFIG_CC2538_RF_OBS_0      tx_active
+#endif
+#ifndef CONFIG_CC2538_RF_OBS_1
+#define CONFIG_CC2538_RF_OBS_1      rx_active
+#endif
+#ifndef CONFIG_CC2538_RF_OBS_2
+#define CONFIG_CC2538_RF_OBS_2      packet_done
```

Comment:
`packet_done` is also always on for me.
I guess it never gets invalidated.
How about `rssi_valid`?